x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/setup_technology.ts

```diff
@@ -180,6 +180,7 @@ export function useSetupTechnology({
           } as NewAgentPolicy;
 
           setNewAgentPolicy(agentlessPolicy);
+          setNewAgentlessPolicy(agentlessPolicy);
```

Comment:
There was an actual issue that was detected thanks to the proper handling of `waitFor`.

https://github.com/user-attachments/assets/b773984e-f30c-4545-9160-5ac072f5ac04

Reply:
After the fix (global_data_tags) included properly on Agentless policies:


https://github.com/user-attachments/assets/f81c021f-88a5-44c3-a6d9-fa1b8a4a7fe0